functions/src/routers/groupRouter.ts

```diff
@@ -122,3 +124,89 @@ groupRouter.post("/add", async (req, res) => {
   // }
   return;
 });
+groupRouter.get("/:userId/groups", async (req, res) => {
+  const uid = req.decodedToken.uid;
+
+  try {
+    // 데이터베이스에서 해당 유저의 정보 가져오기
+    const userData = (
+      await admin.firestore().collection("users").doc(uid).get()
+    ).data();
+
+    // console.log(uid, userData);
+
+    // DB 에서 해당 데이터를 찾을 수 없는경우.
+    if (!userData)
+      return res.status(404).json({ status: 404, error: "USERDATA NOT FOUND" });
+
+    // 해당 유저 그룹에 그룹 정보가 있는지 확인
+    if (!userData.group)
+      return res
+        .status(403)
+        .json({ status: 403, error: "USERGROUP FORBIDDEN", uid });
+
+    // 유저의 그룹 정보 데이터 반환
+    return res.json({
+      status: "200",
+      data: userData.group, // Group ID
+    });
+  } catch (e) {
+    console.error(e);
+    return res.status(500).json({ status: 500, error: "Server error", e });
+  }
+});
+
+groupRouter.post("/:groupId/add/announce", async (req, res) => {
+  const groupId = req.params.groupId.toLowerCase();
+  const {
+    announceName,
+    description,
+    wayofannounce = "email",
+    checked,
+    today,
+  } = req.body;
+  const { uid } = req.decodedToken;
+
+  // 유효성 #1 - 해당 필드들이 존재하여야함.
+  if (!announceName || !description || !wayofannounce || !checked || !today) {
+    return res.status(221).json({
+      status: 221,
+      error: "필수 필드중 하나가 존재하지 않음.",
+    });
+  }
+  // 유효성 #추가??
+  // TODO: memberCount 및 members 정의 후에 해당 요청이 문제가 없을경우
+  // 해당 유저의 정보에 notifs 항목에 해당 그룹을 추가한다.
+  try {
+    const groupdata = (
+      await admin.firestore().collection("groups").doc(groupId).get()
+    ).data();
+    if (!groupdata) {
+      return res.sendStatus(404);
+    }
+    if (!(groupdata.owners && groupdata.owners.includes(uid))) {
+      return res.sendStatus(403);
+    }
+    await admin
+      .firestore()
+      .collection("groups")
+      .doc(groupId)
+      .collection("announcements")
+      .add({
+        announceName,
+        description,
+        wayofannounce,
+        checked,
+        today,
```

Comment:
`admin.firestore.Timestamp.fromDate(new Date());`

```suggestion
        created: admin.firestore.Timestamp.fromDate(new Date()),
```